ddtrace/tracer/spancontext.go

```diff
@@ -322,6 +336,10 @@ func (t *trace) finishedOne(s *span) {
 	if !ok {
 		return
 	}
+	if tr.config.postProcessor != nil && processTrace(t.spans) == false {
```

Comment:
```suggestion
	if !tr.runProcessor(t.spans) {
```
Then you don't need to check inside `processTrace` for a global tracer. It's already there.

Reply:
You can do the `nil` check inside `runProcessor`